docs/ios_device_check.md

```diff
@@ -1,81 +1,158 @@
+1. Enter a name for the key. This is a descriptive name and will not be used
+   in later steps.
 
-1.  Check the box next to "DeviceCheck" to enable using the DeviceCheck with
-    this key.
+1. Check the box next to **DeviceCheck** to enable using the DeviceCheck with
+   this key.
 
-1.  Click "Continue".
+1. Click **Continue**.
 
-1.  On the success page, take **both** of the following actions:
+1. On the confirmation page, do the following:
 
-    1.  Save the DeviceCheck Key ID value to a secure location - this is usually
-        a 10 character alpha-numeric ID like `1BC2D3EFG4`.
+   1. Save the DeviceCheck Key ID value to a secure location - this is usually
+      a 10 character alpha-numeric ID such as `ABCDE1FGHI`.
 
-    1.  Download the DeviceCheck Private Key file (`.p8`) and save it in a
-        secure location. **This file is a private key, so treat it like a
-        password.**
+   1. Download the DeviceCheck Private Key file (with a `.p8` file extension)
+      and save it in a secure location. **This file is a private key, you should
+      treat it like a password.**
 
-
-## Configuring DeviceCheck in the server
+## Configuring the server to use DeviceCheck
 
 To enable and configure Apple's DeviceCheck on the server, you must specify the
-following information for _each_ iOS application your server supports:
+following information for each iOS application your server supports:
 
 - **Team ID** - developer ID, used as the `iss` in the JWT.
 
 - **Key ID** - key ID, used as the `kid` in the JWT.
 
 - **Private Key** - private key (`.p8`), used to sign the JWT.
 
+## Sharing the DeviceCheck key with server operators
+
+You need to share these values with your server operator. If your server is
+operated by a third party, the preferred way to share these secrets is using
+[Secret Manager](https://cloud.google.com/secret-manager/docs/). The secret
+resides in a Google Cloud project that you control, and you can revoke access
+to the secret at any time.
+
+### Prerequisites
+
+- A [Google Cloud account](https://console.cloud.google.com/freetrial) and a
+  [project](https://cloud.google.com/resource-manager/docs/creating-managing-projects).
+
+  When you create a project, keep a note of the project ID, you will need it in
+  this tutorial.
+
+- The [gcloud](https://cloud.google.com/sdk/install) command-line tool.
+
+- Your Team ID, Key ID, and Private Key in .p8 format. These values come from
+  the Apple Developer Portal using the instructions above.
+
+- The server operator's service account email. Your server operator will share
+  this value. This will be of the form `name@project.iam.gserviceaccount.com`.
+
+- A communication channel between yourself and your server operator, such as a
+  shared chat room, email, or a tele/video conference bridge.
+
+### Sharing the private key
+
+To share the DeviceCheck Private Key with a server operator:
+
+1. If you have not already done so, authenticate the gcloud CLI:
+
+   ```console
```

Comment:
These should be `text`, not `console`. sh or bash or console add syntax highlighting which isn't always appropriate and can decrease readability.

Reply:
Thanks, very good point and changed.